pkg/helper/converter/converter.go

```diff
@@ -40,3 +41,20 @@ func ConvertFloat(value any) any {
 		return v
 	}
 }
+
+func AutoConvert(value string) any {
```

Comment:
What are the usages of this function?

Reply:
When setting the variable values, you need to use the corresponding type, or OceanBase will raise an error, this function try to convert the value in string type which is defined in our CRD to the right type.